mozci/task.py

```diff
@@ -401,6 +404,37 @@ def classifications(self):
             and any(result.group == self.name and not result.ok for result in t.results)
         ]
 
+    @property
+    def durations(self):
+        def _subtract(tup):
+            return tup[0] - tup[1]
+
+        durations = []
+
+        for task in self.tasks:
+            data = run_query(
+                "group_summary_by_task_id", Namespace(task_id=task.id, group=self.name)
```

Comment:
We should try to make a single AD query rather than one per task.

If we implement what @ahal suggested in https://github.com/mozilla/mozci/issues/102#issuecomment-604464603:
> To limit queries to ActiveData, we could additionally have a helper that grabs all task durations in a single query and returns a list of `LabelSummaries` (similar to `make_push_objects` in the push.py module.

then this is less important.

Reply:
Agreed, even a query per `GroupSummary` might be too expensive, though we'll need one as a fallback even in the event we implement the idea above. So let's not worry about the idea above for now and tackle it in a follow up PR.

But yeah, for now we should aim for a single query per `GroupSummary` instance (and ensure it gets memoized so we never call it twice).